package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "js-interpreter",
```

Comment:
what about adding a scope, that would allow the distinction to the existing package: `@neil-fraser/js-interpreter` or something along those lines?

https://docs.npmjs.com/misc/scope

Reply:
:+1: on the scoping. Is that @NeilFraser 's npm account?

Reply:
> Is that @NeilFraser 's npm account?

I rather doubt that @NeilFraser has an npm account.  I might be guilty of understatement in describing him as being a bit dubious about the npm ecosystem.  (I still think it would be preferable if he make his packages officially available there, though.)

Reply:
If he registers an account, even if he doesn't publish there, it would be better because we could use a scoped name that no one else could take then

Reply:
>  I might be guilty of understatement in describing him as being a bit dubious about the npm ecosystem

Hopefully things have changed now in 2021 :D